.gitignore

```diff
@@ -1,54 +1,54 @@
+*-pak
```

Comment:
The use of .gitignore files per directory was a deliberate choice. Ignored files that are specific to that directory are stored locally, also simplifying the root .gitignore. (And if we were to solely use a root .gitignore, the directory specific files would need a directory specifier to avoid ignoring necessary files.)

The current ordering in .gitignore groups related files. That is lost when sorted purely lexicographically.

All that said, the .gitignore files do need cleaning up. Long gone files are still ignored and some, but only some, directory specific files are already in the root .gitignore. So I do support cleanup that leaves the .gitignore files in the subdirectories, where appropriate.

Reply:
This makes total sense I shall revert the .gitignores to how they were before, maybe with a little bit or cleaning if anything is obviously out of place.